internal/auth0/organization/resource_members.go

```diff
@@ -0,0 +1,203 @@
+package organization
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/google/go-cmp/cmp"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/auth0/terraform-provider-auth0/internal/config"
+	"github.com/auth0/terraform-provider-auth0/internal/value"
+)
+
+// NewMembersResource will return a new auth0_organization_members (1:many) resource.
+func NewMembersResource() *schema.Resource {
+	return &schema.Resource{
+		Description:   "This resource is used to manage members of an organization.",
+		CreateContext: createOrganizationMembers,
+		ReadContext:   readOrganizationMembers,
+		UpdateContext: updateOrganizationMembers,
+		DeleteContext: deleteOrganizationMembers,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"organization_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The ID of the organization to assign the member to.",
```

Comment:
```suggestion
				Description: "The ID of the organization to assign the members to.",
```

Reply:
Updated in [`9d1583d` (#614)](https://github.com/auth0/terraform-provider-auth0/pull/614/commits/9d1583d60fcd55406bd65a22b4bab6e63384b907)